sagemaker-python-sdk/1P_kmeans_highlevel/kmeans_mnist.ipynb

```diff
@@ -111,18 +113,20 @@
    "outputs": [],
    "source": [
     "import matplotlib.pyplot as plt\n",
-    "plt.rcParams[\"figure.figsize\"] = (2,10)\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = (2, 10)\n",
     "\n",
     "\n",
-    "def show_digit(img, caption='', subplot=None):\n",
-    "    if subplot==None:\n",
-    "        _,(subplot)=plt.subplots(1,1)\n",
-    "    imgr=img.reshape((28,28))\n",
-    "    subplot.axis('off')\n",
-    "    subplot.imshow(imgr, cmap='gray')\n",
+    "def show_digit(img, caption=\"\", subplot=None):\n",
+    "    if subplot == None:\n",
```

Comment:
Please change this to `if subplot is None:`

Reply:
Changed.